### DISCUSSION TITLE
Greenify with LSPosed

### DISCUSSION THREAD

**Discussion by NichirinSword** [LSPosed/LSPosed#1599] [Q&A]
What should I tick on greenify module in Isposed to activate? I am confused because Lsposed does not select any app for greenify module by default. Which one should I tick.

I am using Android 9 with OneUI 1.1 and Latest Magisk 24.1 and latest Zygisk - LSPosed build from CI build telegram channel

**Comment by WorldofShadows** [comment]
Hey,
had the same problem. I ticked "System Framework" rebooted the device and the Greenify worked again without crashing. 
But this is on LineageOS 18.1 / Android 11 /w latest magisk and Zygisk, maybe that works on Android 9, too.

**Comment by FrankHB** [comment]
"System Framework" ticked works in EMUI 9.0 (Android 9.0).

**Comment by Zaxx420** [comment]
I'm rooted with the latest Magisk w/ LSposed (Zygisk) running stock A12 w/BlueSpark kernel on my Pixel 4a.  When I add the 'greenify4magisk' module and tick the exposed option in greenify all seems well until I reboot. About 7-8 sec. in (prolly when it loading greenify) the screen freezes up for like 60 sec. (it times-out maybe) then reboots into a dreaded bootloop. It's easy to fix cuz I can uninstall things b4 it freezes then next boot all is good. Is there maybe a certain order of installation maybe? I dunno...has anyone else tried greenify with A11/12 maybe? I know there are other apps for hibernation and aggressive doze but I got the paid ver. back in A9 (2 phones ago...lol) and it has a few extras I like. Tia...

**Comment by qwq233** [comment]
its not LSPosed's issue
ask your module author
